internal/storage/image.go

```diff
@@ -472,31 +472,36 @@ func copyImageChild() {
 		os.Exit(1)
 	}
 
-	if err := moveSelfToCgroup(args.ParentCgroup, args.HasCollectMode); err != nil {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	if err := moveSelfToCgroup(ctx, args.ParentCgroup, args.HasCollectMode); err != nil {
 		fmt.Fprintf(os.Stderr, "%v", err)
 		os.Exit(1)
 	}
 
```

Comment:
could we do `defer cancel()`

Reply:
Not any more since I changed the PR to not touch this code section again. :)